src/mame/sega/suprloco.cpp

```diff
@@ -535,13 +688,18 @@ ROM_START( suprloco )
 	ROM_LOAD( "epr-5230.56",     0x4000, 0x2000, CRC(f04a4b50) SHA1(80363f0c508fb2a755bf684f9a6862c1e7285495) )
 							// 0x6000 empty
 
-	ROM_REGION( 0x0620, "proms", 0 )
-	ROM_LOAD( "pr-5220.100",     0x0100, 0x0080, CRC(7b0c8ce5) SHA1(4e1ea5ce38198a3965dfeb609ba0c7e8211531c3) ) // color PROM
-	ROM_CONTINUE(                0x0000, 0x0080 )
-	ROM_CONTINUE(                0x0180, 0x0080 )
-	ROM_CONTINUE(                0x0080, 0x0080 )
-	ROM_LOAD( "pr-5219.89",      0x0200, 0x0400, CRC(1d4b02cb) SHA1(00d822f1bc4f57f2f5d5a0615241f8136246a842) ) // 3bpp to 4bpp table
-	ROM_LOAD( "pr-5221.7",       0x0600, 0x0020, CRC(89ba674f) SHA1(17c87840c8011968675a5a6f55966467df02364b) ) // unknown
+	ROM_REGION( 0x0400, "colorprom", 0 )
+	ROM_LOAD( "pr-5220.100",     0x0000, 0x0200, CRC(7b0c8ce5) SHA1(4e1ea5ce38198a3965dfeb609ba0c7e8211531c3) ) // color PROM, 82S141 512x8
+	ROM_RELOAD(0x200, 0x200) // The socket can hold an 82S181 1kx8 prom but is not used here, so repeat the prom since pin 22 is N/C on an 82S141
+	ROM_REGION( 0x0400, "clutprom", 0 )
+	ROM_LOAD( "pr-5219.89",      0x0000, 0x0400, CRC(1d4b02cb) SHA1(00d822f1bc4f57f2f5d5a0615241f8136246a842) ) // 3bpp to 4bpp Color LUT
+	ROM_REGION( 0x0020, "proms", 0 )
+	ROM_LOAD( "pr-5221.7",       0x0000, 0x0020, CRC(89ba674f) SHA1(17c87840c8011968675a5a6f55966467df02364b) ) // tilemap palette latch timing
+
+	// Two undumped PALS:
+	// PAL16R4@IC20: horizontal counter carry, CSYNC generation, and sprite DMA timing
+	// PAL16R4@IC30: vertical counter handling, maincpu INT generation, and VSYNC timing
+	// These are NOT the same as the two pals in System1, and some of the functions are switched around there!
```

Comment:
“PALs”, not “PALS” or “pals”.  Also “System 1”.

Reply:
This should be fixed now.